src/main/java/uk/gov/hmcts/befta/player/DefaultBackEndFunctionalTestScenarioPlayer.java

```diff
@@ -621,12 +631,53 @@ private void authenticateUser(final BackEndFunctionalTestScenarioContext scenari
             UserData user) {
         String logPrefix = scenarioContext.getTestData().get_guid_() + ": " + prefix + " [" + user.getUsername() + "] ";
         String preferredTokenProviderClientId = scenarioContext.getTestData().getUserTokenClientId();
-        scenario.log("Authentication attempt from: " + preferredTokenProviderClientId + ".");
-        try {
+
+        AuthenticationRetryConfiguration authenticationRetryConfiguration =
+                BeftaMain.getConfig().getAuthenticationRetryConfiguration();
+
+        if (authenticationRetryConfiguration.isRetryDisabled()) {
+            try {
+                scenario.log("Authentication attempt from: " + preferredTokenProviderClientId + ".");
```

Comment:
Please take this line above if(..) statement.

Reply:
moved the line